.gitignore

```diff
@@ -5,3 +5,6 @@
 /assets/completions/bat.fish
 /assets/manual/bat.1
 /assets/metadata.yaml
+
+# IDEA config files
+.idea/
```

Comment:
Can we please remove this? I think it rather belongs in a user git ignore file, not into this repo.

Reply:
Sure.